uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrls.spec.js

```diff
@@ -32,6 +32,7 @@
       var date2017 = '02/02/2017';
       var date2013 = '02/02/2013';
       var dateServer2017 = '2017-02-02';
+      var role = 'staff'; // change this value to set other roles
```

Comment:
I am not sure anyone will change vars manually for testing. Tests have to run automatically for all roles in one go.  I know I introduced the same solution as per https://github.com/civicrm/civihr/pull/1977/files but I just realised, that might not be ideal. Lets have a call @deb1990 and discuss.